src/store/metadata/stub/content/datasets.json

```diff
@@ -47,8 +47,54 @@
                 "modified": "2017-02-21T09:30:00Z"
               }
             ]
-        }
+        },
+        {
+          "@id": "https://staging.idpd.uk/datasets/a16mu",
+          "@type": "dcat:DatasetSeries",
+          "identifier": "a16mu",
+          "title": "Modelled unemployment rate aged 16 years and over",
+          "summary": "Modelled unemployment rate for people aged 16 years and over, Great Britain, 07/2004-06/2005 to 07/2022-06/2023.",
+          "description": "This dataset shows modelled-based estimates for the proportion of people aged 16 years and over without a job who have been actively seeking work within the last four weeks and are available to start work within the next two weeks",
+          "issued": "2023-11-06T12:25:28",
+          "modified": "2023-11-06T12:25:28",
+          "next_release": "2024-11-06T12:00:00",
+          "publisher": "https://staging.idpd.uk/publishers/office-for-national-statistics",
+          "creator": "https://www.gov.uk/government/organisations/department-for-education",
+          "contact_point": {
+            "name": "Debra Leaker",
+            "email": "mailto:labour.market@ons.gov.uk"
+          },
+          "topics": [
+            "https://staging.idpd.uk/topics/employment",
+            "https://staging.idpd.uk/topics/labour-market-and-welfare"],
```

Comment:
these will need to match the topics you're listing at the edition level.

RE topics, are are going to need them (otherwise we end up with broken links on the api, but you don't need to get into the weeds on parents and children unless you want to, as an example a topic of

```
            "@id": "https://staging.idpd.uk/topics/some-topic",
            "@type": "dcat:theme",
            "identifier": "some-topic",
            "title": "Some Topic",
            "description": "something describing some topic",
            "sub_topics": [],
            "parent_topics": []
        },
```

is completely viable for now, we can always get into parents/children as a separate piece later on.

Reply:
kept only the topic referenced in an edition, and added the entry in topics.json